packages/mobile/src/invite/saga.ts

```diff
@@ -182,15 +183,32 @@ export function* sendInvite(
     const inviteCode = createInviteCode(temporaryWalletAccount.privateKey)
     const name = yield select(nameSelector)
 
-    const link = yield call(generateInviteLink, inviteCode)
-    const message = i18n.t(
-      amount ? 'sendFlow7:inviteWithEscrowedPayment' : 'sendFlow7:inviteWithoutPayment',
-      {
-        name,
-        amount: amount?.toString(),
-        link,
+    const link = features.ESCROW_WITHOUT_CODE
+      ? VALORA_DOMAIN
+      : yield call(generateInviteLink, inviteCode)
+
+    let messageProp
+    if (features.ESCROW_WITHOUT_CODE && amount) {
+      messageProp = 'sendFlow7:inviteWithEscrowAndNoCode'
+    } else if (amount) {
+      messageProp = 'sendFlow7:inviteWithEscrowedPayment'
+    } else {
+      messageProp = 'sendFlow7:inviteWithoutPayment'
+    }
+
+    const message = i18n.t(messageProp, {
+      name,
+      amount: amount?.toString(),
+      link,
+    })
+
+    if (features.ESCROW_WITHOUT_CODE) {
+      if (amount) {
+        yield call(initiateEscrowTransfer, e164Number, amount)
       }
-    )
+      yield call(navigateToInviteMessageApp, e164Number, inviteMode, message)
```

Comment:
This `navigateToInviteMessageApp` uses SMS by default, not the native share. We can just do `yield call(Share.share, { message })` to open the native share dialog and let users choose how to share.